### DISCUSSION TITLE
Adding subfolders in web interface

### DISCUSSION THREAD

**Discussion by ninjamonkey198206** [dani-garcia/vaultwarden#2358] [Q&A]
Am I missing something? Is there a simple method of doing this?

This and a few other QoL interface functions that seem to be missing are the only complaints I get from users.

**Comment by jjlin** [comment]
See https://bitwarden.com/help/folders/#nested-folders

**Comment by ninjamonkey198206** [comment]
Yeah, that's just a bit cumbersome to a lot of users.

I was thinking of a menu option to add a folder. I don't know how complex that would be to code, but it would make it easier to use for most end users.

**Comment by jjlin** [comment]
Vaultwarden only implements the server side; it relies completely on the upstream Bitwarden clients.

You can check https://community.bitwarden.com/c/feature-requests/5/ for any similar requests and upvote them, or else open your own feature request there.

**Reply by ninjamonkey198206** [reply]
Ah, my bad. I'll go browse through there. Thank you.
